node/service/src/lib.rs

```diff
@@ -718,6 +718,13 @@ where
 	Ok(leaves.into_iter().rev().take(MAX_ACTIVE_LEAVES).collect())
 }
 
+pub fn create_availability_config() -> AvailabilityConfig {
+	AvailabilityConfig {
+		col_data: parachains_db::REAL_COLUMNS.col_availability_data,
+		col_meta: parachains_db::REAL_COLUMNS.col_availability_meta,
+	}
+}
```

Comment:
Maybe a `const` would be more appropriate rather than a function? I don't have a strong opinion here; just something for you to consider.